index.js

```diff
@@ -1,7 +1,7 @@
 var path = require('path');
 var pkg = require('./package.json');
 
-var jeetPath = path.join(__dirname, 'styl')
+var jeetPath = path.join(__dirname, 'styl');
```

Comment:
Can you get rid of these [semi-colons](https://github.com/feross/standard#the-rules)?